nuxeo-dm/nuxeo-platform-webapp-core/src/test/java/org/nuxeo/ecm/lifeCycle/BulkLifeCycleChangeListenerTest.java

```diff
@@ -116,4 +116,42 @@ public void testCopyLifeCycleHandler() {
         }
     }
 
+    /*
+     * NXP-22608
+     */
+    @Test
+    public void testLifeCycleAPITwoLevels() {
+
+        DocumentModel folderDoc1 = session.createDocumentModel("/", "testFolder1", "Folder");
+        folderDoc1 = session.createDocument(folderDoc1);
+        DocumentModel folderDoc2 = session.createDocumentModel(folderDoc1.getPathAsString(), "testFolder2", "Folder");
```

Comment:
This pattern has been used for a long time in old tests, but it's better to use an explicit `"/testFolder1"` rather than `folderDoc1.getPathAsString()`. It's just more readable.